libraries/AP_InertialSensor/AP_InertialSensor.h

```diff
@@ -233,7 +233,9 @@ class AP_InertialSensor : AP_AccelCal_Client
 
     // get the accel filter rate in Hz
     uint16_t get_accel_filter_hz(void) const { return _accel_filter_cutoff; }
-
+private:
+    bool externally_temperature_calibrated = false;
```

Comment:
This is almost certainly in the wrong class.  This should be in AP_InertialSensor_Backend.